### ISSUE TITLE
All in One WP Security compatibility: 403 errors on ongoing fetch requests

### ISSUE THREAD

**Issue by ernee** [google/site-kit-wp#917]
As reported in this [forum topic](https://wordpress.org/support/topic/sitekit-no-working-no-data/), the [All in One WP Security](https://wordpress.org/plugins/all-in-one-wp-security-and-firewall/) plugin option for [6G Firewall protection](https://perishablepress.com/6g/) blocks malicious URLs and disrupts Site Kit's ongoing fetch requests preventing data from displaying in the dashboard. Disabling the option appears to restore dashboard data. Similar to #884 

1. Install and activate [All in One WP Security](https://wordpress.org/plugins/all-in-one-wp-security-and-firewall/)
2. Enable _WP Security > Firewall > 6G Blacklist Firewall Rules > Enable 6G Firewall Protection_
3. Go to the Site Kit Dashboard and observe missing data on the dashboard, console errors in the browser.

## Screenshots
<img width="1253" alt="All in One WP Security 403 error" src="https://user-images.githubusercontent.com/8108643/69675109-c19ee480-1052-11ea-9d08-7c6ed4d05dad.png">

## Additional Context

- PHP Version: 7.1.33
- OS: MacOS
- Browser: Chrome
- Plugin Version: 1.1.1
- Device: MacBook Air

Below is a typical url being blocked:
```
https://www.getbeans.io/wp-json/google-site-kit/v1/data/?request=[{"data":{"dateRange":"last-28-days","dimensions":"ga:medium","metrics":[{"expression":"ga:sessions","alias":"Sessions"},{"expression":"ga:users","alias":"Users"},{"expression":"ga:newUsers","alias":"New Users"}],"orderby":[{"fieldName":"ga:sessions","sortOrder":"DESCENDING"}],"limit":10,"url":false},"type":"modules","identifier":"analytics","datapoint":"report","priority":1,"maxAge":86400,"context":["Dashboard","Single"],"key":"modules::analytics::report::1cc2cfc3218eab9aa0d167f96fe944ea"},{"data":{"dateRange":"last-28-days","dimensions":"ga:medium","metrics":[{"expression":"ga:sessions","alias":"Sessions"},{"expression":"ga:users","alias":"Users"},{"expression":"ga:newUsers","alias":"New Users"}],"orderby":[{"fieldName":"ga:sessions","sortOrder":"DESCENDING"}],"limit":10},"type":"modules","identifier":"analytics","datapoint":"report","priority":1,"maxAge":86400,"context":"Dashboard","key":"modules::analytics::report::e54ff2cbd79c08e767fbf3260cc933bb"},{"data":{"dateRange":"last-28-days","url":false,"dimensions":"date","compareDateRanges":true},"type":"modules","identifier":"search-console","datapoint":"searchanalytics","priority":1,"maxAge":86400,"context":["Single","Dashboard"],"key":"modules::search-console::searchanalytics::6892af84f7b8398712c112e420a28141"},{"data":{"dateRange":"last-28-days","multiDateRange":1,"dimensions":"ga:date","metrics":[{"expression":"ga:users","alias":"Users"},{"expression":"ga:sessions","alias":"Sessions"},{"expression":"ga:bounceRate","alias":"Bounce Rate"},{"expression":"ga:avgSessionDuration","alias":"Average Session Duration"},{"expression":"ga:goalCompletionsAll","alias":"Goal Completions"},{"expression":"ga:pageviews","alias":"Pageviews"}],"limit":10,"url":false},"type":"modules","identifier":"analytics","datapoint":"report","priority":1,"maxAge":86400,"context":"Dashboard","key":"modules::analytics::report::ab522b830a0f488c2585f21b935d1798"},{"data":{"dateRange":"last-28-days","compareDateRanges":1,"dimensions":"ga:date","metrics":[{"expression":"ga:users","alias":"Users"},{"expression":"ga:sessions","alias":"Sessions"},{"expression":"ga:bounceRate","alias":"Bounce Rate"},{"expression":"ga:avgSessionDuration","alias":"Average Session Duration"},{"expression":"ga:goalCompletionsAll","alias":"Goal Completions"}],"limit":180,"url":false},"type":"modules","identifier":"analytics","datapoint":"report","priority":1,"maxAge":86400,"context":"Dashboard","key":"modules::analytics::report::743969d0dab246e5788b60b01f73ea7e"},{"data":{"dateRange":"last-28-days","url":false},"type":"modules","identifier":"analytics","datapoint":"goals","priority":1,"maxAge":3600,"context":"Dashboard","key":"modules::analytics::goals::a096c9c425bdbc7e6487d28f65a7bdf5"},{"data":{"dateRange":"last-28-days","dimensions":"query","limit":10},"type":"modules","identifier":"search-console","datapoint":"searchanalytics","priority":1,"maxAge":86400,"context":["Dashboard"],"key":"modules::search-console::searchanalytics::d5590576de7dfd3412f70ad0ee9bdfc8"},{"data":{"dateRange":"last-28-days","dimensions":"ga:pageTitle,ga:pagePath","metrics":[{"expression":"ga:pageviews","alias":"Pageviews"},{"expression":"ga:uniquePageviews","alias":"Unique Pageviews"},{"expression":"ga:bounceRate","alias":"Bounce rate"}],"orderby":[{"fieldName":"ga:pageviews","sortOrder":"DESCENDING"}],"limit":10},"type":"modules","identifier":"analytics","datapoint":"report","priority":1,"maxAge":86400,"context":["Single","Dashboard"],"key":"modules::analytics::report::b9d29b5a688b67a72c16f5cd56b71aae"}]&_locale=user
```

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* Replace the use of long query parameters which is causing conflict with some security measurement.

## Implementation Brief

* 

## Changelog entry

*

**Comment by aaemnnosttv** [comment]
I wasn't able to reproduce this locally since this plugin seems to integrate with htaccess, but the defined AC is already addressed by #779.

**Comment by ThierryA** [comment]
This is fixed by #779 which will be in the next release.
